File: LABORATORIOS/LABORATORIO2/Ejercicio6/Python/Secuencia.py
def calcular_suma_secuencia(n):
    suma = 0
    multiplicador = 1

    for i in range(1, n + 1):
        termino = i * multiplicador
        suma += termino
        multiplicador *= -2

    return suma

n_terminos = 6
suma_secuencia = calcular_suma_secuencia(n_terminos)
print(suma_secuencia)

File: LABORATORIOS/LABORATORIO2/Ejercicio6/Java/Secuencia.java
public class Secuencia {
    public static int calcularSumaSecuencia(int n) {
        int suma = 0;
        int multiplicador = 1;

        for (int i = 1; i <= n; i++) {
            int termino = i * multiplicador;
            suma += termino;
            multiplicador *= -2;
        }

        return suma;
    }

    public static void main(String[] args) {
        int nTerminos = 6;
        int sumaSecuencia = calcularSumaSecuencia(nTerminos);
        System.out.println(sumaSecuencia);
    }
}